### ISSUE TITLE
Lists, scrolling, and "+"

### ISSUE THREAD

**Issue by eyesurewould** [buildabeer/buildabeer#55]
I find it quite difficult to scroll around the various lists in the app because each list container receives its own scroll bar (since it doesn't automatically grow to fit the list). I suggest these containers be resized to avoid scrollbars.

In addition, items in these lists use the "+" (plus) to indicate there is more detail for the user to view. Generally, the "+" sign means "make a new [whatever]" (e.g. 'add something').  Therefore, I suggest changing the "+" to a 'down caret' as used in a [Material Design expandable list](https://material.io/design/components/lists.html#types) - see the "Expand and Collapse" section.

![](https://storage.googleapis.com/spec-host-backup/mio-design%2Fassets%2F1US53IsyABmg3sFnHk2sROcI8qgNgGkMk%2Flists-controls-expandable-open.png)

**Comment by buildabeer-admin** [comment]
I agree wholeheartedly. This would be much more clear. This would appear in all of the index pages as well as the recipe designer, recipe runner, and (maybe) the recipe viewer.
